appveyor.yml

```diff
@@ -65,6 +65,12 @@ before_deploy:
   - C:\venv\Scripts\activate
   - python --version
   - pip install -r requirements/deploy.txt
+  - ps: >-  
```

Comment:
Were you able to test this block? Or drop it in a yaml formatter and then check the resulting string? I'm suspecting we may need semicolons after each line to separate the commands properly...

Reply:
In S3, watchmaker-dev/releases, the latest build from my fork is 0.9.7.dev73727f6. The AppVeyor and Travis files are living together in perfect harmony.